### PR TITLE
CB-9504 Replace FreeIPA error page

### PR DESCRIPTION
**Pull request by daszabo**
See detailed description in the commit message.

### CODE REVIEW

**Review by pdarvasi — APPROVED** [review]

File: freeipa/src/main/resources/freeipa-salt/salt/freeipa/patch-error-page.sls [hunk 1]
```diff
@@ -0,0 +1,24 @@
+restart_ipa:
+  service.running:
+      - name: ipa
```
Comment (lacikaaa) [inline freeipa/src/main/resources/freeipa-salt/salt/freeipa/patch-error-page.sls:3]: I think restarting `pki-tomcatd@pki-tomcat.service` should be enough

File: freeipa/src/main/resources/freeipa-salt/salt/freeipa/patch-error-page.sls [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - watch:
+        - file: /var/lib/pki/pki-tomcat/conf/web.xml
+
+replace_default_tomcat_error_page:
+  file.replace:
+    - name: /var/lib/pki/pki-tomcat/conf/web.xml
+    - pattern: '</web-app>'
+    - repl: "<error-page>
+             <error-code>404</error-code>
+                <location>/error.jsp</location>
```
Comment (lacikaaa) [inline freeipa/src/main/resources/freeipa-salt/salt/freeipa/patch-error-page.sls:13]: is this page exists?

**Review by lacikaaa — CHANGES_REQUESTED** [review]
replicas are missing this fix

**Comment by pdarvasi** [comment]
@daszabo could you finish the testing or would you need some help? Thanks

**Comment by daszabo** [comment]
Tested with FreeIPA HA, it works properly. Also made sure multiple runs yields the same result.

**Review by lacikaaa — APPROVED** [review]

**Pull request merged**